NOTICE.txt

```diff
@@ -4511,7 +4511,7 @@ THE SOFTWARE.
 
 
 grpcio
-1.69.0
```

Comment:
This doesn't look right

Reply:
I checked it and we need to re-release python-elastic-agent-client, will do today

Reply:
Should be fixed by https://github.com/elastic/connectors/pull/3150